Web/Registration.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE >
+<html>
+	<head>
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="Styles_registration.css" type="text/css">
+		<script type="text/javascript" src="registration.js"></script>
+		<title>Registration</title>
+	</head>
+	
+	<body>
+		<header>
+			Registration
+		</header>
+		<div class="main">
+			<form name="registration" id="registration" onSubmit="return validate();">
+			<article>
+				First Name:
+				</br>
+				</br>
+				Last Name:
+				</br>
+				</br>
+				Password:
+				</br>
+				</br>
+				Re-enter Password:
+				</br>
+				</br>
+				Age:
+				</br>
+				</br>
+				State:
+				</br>
+				</br>
+				City:
+				</br>
+				</br>
+				Address Line 1:
+				</br>
+				</br>
+		    </article>
+			<section>
+				<input type="text" name="first_name" id="first_name" required auto focus placeholder="First Name"/>
+				</br></br>
+				<input type="text" name="last_name" id="last_name" required auto focus placeholder="lastName"/>
+				</br></br>
+				<input type="password" name="password" id="password" required auto focus placeholder="Enter Password"/>
+				</br></br>
+				<input type="password" name="re_password" id="re_password" required auto focus placeholder="re-enter password"/>
+				</br></br>
+				<input type="text" name="age" id="age" required auto focus placeholder="enter age"/>
```

Comment:
ACTION: Age (Must be numeric). It should not allow user to type any character other the 0-9 and DOT (.)